Code/Framework/AzToolsFramework/AzToolsFramework/Entity/ReadOnly/ReadOnlyEntitySystemComponent.cpp

```diff
@@ -42,6 +42,11 @@ namespace AzToolsFramework
 
     bool ReadOnlyEntitySystemComponent::IsReadOnly(const AZ::EntityId& entityId)
     {
+        if (!entityId.IsValid())
+        {
+            return false;
```

Comment:
I wonder if 'false' is the right output for "IsReadyOnly' but I assume that this would be a indicator 'this needs to be filled out' maybe?

Reply:
Well this kind of assumes that false is the default. To be fair this is mostly a safety check so that we don't crash if some system calls this API with invalid data, the actual value returned is undefined behavior.